TestConfig/scripts/testKitGen/makeGenTool/mkgen.pl

```diff
@@ -458,7 +460,7 @@ sub specToPlatGen {
 		if ( defined $sp_hs->{$graphSpec} ) {
 			$spec2platform .= "ifeq" . " (\$(SPEC),$graphSpec)\n\tPLATFORM=" . $sp_hs->{$graphSpec}->{"platform"} . "\nendif\n\n";
 		} else {
-			print "\nWarning: cannot find spec $graphSpec in ModesDictionaryService or ottawa.csv file!\n";
+			print "\nWarning: cannot find spec $graphSpec in ModesDictionaryService or ottawa.csv file!\n" if DEBUG;
```

Comment:
This ! does not immediately follow a file name, so I thought it would not cause much confusion. But, I can remove it in the next commit anyway.